lib/outdated.js

```diff
@@ -85,7 +86,7 @@ class Outdated extends ArboristWorkspaceCmd {
     }))
 
     // sorts list alphabetically
-    const outdated = this.list.sort((a, b) => a.name.localeCompare(b.name, 'en'))
+    const outdated = this.list.sort(({ name: a }, { name: b }) => localeCompare(a, b)
```

Comment:
```suggestion
    const outdated = this.list.sort(({ name: a }, { name: b }) => localeCompare(a, b))
```

Reply:
Ha, fixed that one locally, but neglected to add it to my amended commit.  Thanks.